### DISCUSSION TITLE
How can I edit themes in Docker version?

### DISCUSSION THREAD

**Discussion by DearFox** [shish/shimmie2#1279] [Q&A] [answered]
I tried to search in different folders, and in folders with docker - but it seems I could not find where exactly everything is located.
I have access to the main folders with pictures and the database, I believe, but I couldn't find the folder with themes and other files that I would like to customize.

**Comment by shish** [comment, marked as answer]
I've added some documentation to https://github.com/shish/shimmie2/wiki/Docker#adding-custom-themes - though that still assumes you have a theme already. Creating it in the first place is a little trickier, since "copy-paste an existing theme and tweak it" is harder when you don't have direct access to the files (though worst case you could download the .zip version of shimmie to see what themes look like there and then copy one of those)
